ddtrace/internal/datadog/profiling/dd_wrapper/src/sample_manager.cpp

```diff
@@ -0,0 +1,51 @@
+#include "sample_manager.hpp"
+#include "types.hpp"
+
+using namespace Datadog;
+
+void
+SampleManager::add_type(SampleType type)
```

Comment:
Does it ever make sense to add a type to an existing `SampleManager`

Reply:
This is something we'll want to support for dynamic instrumentation, but currently the interface expects initialization to occur only once.